### PR TITLE
data:ParasGoswami06

### PR DESCRIPTION
**Pull request by ParasGoswami06**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4284"></a>

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

File: data/ParasGoswami06.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/parasgoswami243/?igshid=YmMyMTA2M2Y%3D",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/paras-goswami-b98658210/",
+      "icon": "FaLinkedin"
+    }
```
Comment (swagataroy30) [inline data/ParasGoswami06.json:21]:
```suggestion
    },
```

**Review by swagataroy30 — COMMENTED** [review]
you need to add a comma at line 21. Thank You!

**Pull request merged**